de.fu_berlin.inf.dpp.intellij/src/de/fu_berlin/inf/dpp/intellij/ui/wizards/AddProjectToSessionWizard.java

```diff
@@ -5,10 +5,13 @@
 import java.io.IOException;
 import java.nio.file.Path;
 import java.nio.file.Paths;
+import java.util.ArrayList;
```

Comment:
![Code Smell](https://sonarsource.github.io/sonar-github/code_smell.png 'Code Smell') Code Smell: Remove this unused import 'java.util.ArrayList'. ([squid:UselessImportCheck](https://sonarcloud.io/organizations/saros/rules#rule_key=squid%3AUselessImportCheck))

[See it in SonarCloud](https://sonarcloud.io/project/issues?id=saros_complete&pullRequest=273&issues=AWaWrfeQsSPV6ZXI_B9R&open=AWaWrfeQsSPV6ZXI_B9R)

[ISSUE_KEY]: AWaWrfeQsSPV6ZXI_B9R